lib/Jsrt/Core/JsrtContextCore.h

```diff
@@ -138,7 +138,7 @@ class ChakraCoreHostScriptContext sealed : public HostScriptContext
 
     HRESULT SetDispatchInvoke(Js::JavascriptMethod dispatchInvoke) override
     {
-        AssertMsg(false, "no hostdispatch in jsrt");
+        // AssertMsg(false, "no hostdispatch in jsrt");
         return E_NOTIMPL;
```

Comment:
Why this change? I had fixed a similar issue with my JsRTDebugging change where I initialized DispatchProfileInvoke in  ScriptContext and check for in ScriptContext::SetDispatchProfile.